stream-legal-entity/legal-entity-core/src/main/java/com/backbase/stream/LegalEntitySaga.java

```diff
@@ -902,7 +902,7 @@ private Mono<LegalEntityTask> processSubsidiaries(LegalEntityTask streamTask) {
 
         Flux<LegalEntity> subsidiaries = parentLegalEntity.getSubsidiaries() == null
             ? Flux.empty()
-            : Flux.fromIterable(parentLegalEntity.getSubsidiaries());
+            : Flux.fromIterable(parentLegalEntity.getSubsidiaries().stream().filter(x -> x.getCreateInDBS()).toList());
```

Comment:
why to use the `.toList()` here? the stream itself is iterable.

Reply:
Stream is not iterable so it is needed.

Reply:
You can replace it with `Flux.fromStream` so you don't need to do unnecessary transformations

Reply:
thanks done. please check now.